soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -1113,7 +1113,7 @@ void FileChoose_DrawWindowContents(GameState* thisx) {
 
         // draw rando label
         if (Save_GetSaveMetaInfo(i)->randoSave) {
-            if (CVar_GetS32("gHudColors", 1) == 2) {
+            if (CVar_GetS32("gHudColors", 1) == 2 && FileChoose_IsSaveCompatible(Save_GetSaveMetaInfo(1))) {
```

Comment:
why is a hardcoded `1` being passed into `Save_GetSaveMetaInfo` here?

Reply:
Oh that must be a typo, should be an i